portfolio/src/main/webapp/comment.js

```diff
@@ -61,12 +69,13 @@ function loadAndShowComments() {
  */
 function getFormattedComment(json) {
     let commenter = json.commenter;
+    let email = json.email;
     let content = json.content;
     let timestamp = json.timestamp;
 
     let timeString = getFormattedDate(timestamp);
 
-    let resultString = `Commenter: ${commenter}\nTime: ${timeString}\nComment: ${content}`;
+    let resultString = `Commenter: ${commenter}\nEmail: ${email}\nTime: ${timeString}\nComment: ${content}`;
```

Comment:
Optionally
```
let fieldNameValueMap = {
    "Commenter": commenter, 
    "Email": email, 
    "Time": timeString, 
    "Comment": comment" }
let resultString = some simple conversion from the map to a string
```
Could use array of array of size 2 as well, whatever works and easier to read.

Reply:
Ahh I think the original is simpler; i will reformat it to look cleaner